inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -710,7 +710,8 @@ V10Parser::LayerBaseCreator::fillSubGraphLayer(const ngraph::OutputVector &input
     for (const auto& input : input_map) {
         auto &_input = input.second;
         auto axis_attr = _input.attribute("axis");
-        size_t ti_input_index = GetUIntAttr(_input, "external_port_id");
+        auto purpose = GetStrAttr(_input, "purpose", "");
+        int64_t ti_input_index = GetInt64Attr(_input, "external_port_id");
```

Comment:
Please, fix line 704 as well. If you don't change it then I cannot read IR files with "execution_condition" output.

Reply:
fixed